pymilvus_orm/index.py

```diff
@@ -57,12 +57,10 @@ def __init__(self, collection, field_name, index_params, name="", **kwargs):
         self._kwargs = kwargs
 
         conn = self._get_connection()
-        index = conn.describe_index(self._collection.name, "")
-        if index is None:
+        index = conn.describe_index(self._collection.name)
+        tmp_field_name = index.pop("field_name")
```

Comment:
better to pass in a default value when calling `pop` to avoid throwing exceptions.